### DISCUSSION TITLE
during running fepx example file1 for large strain

### DISCUSSION THREAD

**Discussion by Surajit94** [neperfepx/FEPX#73] [Q&A] [answered]
when i run the FEPX example file1 for large strain my mesh are very distorted. how i fix that problem ? i am doing grain generation and meshing by NEPER 4.3.2-4.

**Comment by mkasemer** [comment, marked as answer]
What do you mean by large strain and very distorted? One would expect large distortion at a large strain.

Do you mean the visualizations? Note that the visualizations produced by Neper utilize a scaling of 10x to exaggerate the deformation field. If you want the deformed mesh to be more realistically plotted, you can reduce or remove this scaling.

**EDIT** by @rquey: The 10x exaggeration is only for the examples provided with FEPX, not in general (see [-datanodecoofact](https://neper.info/doc/neper_v.html#cmdoption-datanodecoofact)).

**Reply by mkasemer** [reply]
Sorry, I should be more clear that _the examples provided with FEPX, and specifically the first example_ visualizes the deformation with 10x exaggeration. @rquey is correct that the default is no scaling.

If you are using example 1 as a template, the deformed mesh that you plot will be greatly exaggerated.

**Reply by Surajit94** [reply]
First of all thank you  for your suggestion. 
What will be the best example option to visualize strain like 10% to 30% for FCC  with strain rate of .01.? and also how i can visualize
the deformation in some time interval till the final deformation?

**Reply by mkasemer** [reply]
@Surajit94 --- visualization is up to you, and dependent on what you are trying to show. 10x exaggeration is extreme, and is only meant to illustrate the way in which deformation is proceeding with the chosen boundary conditions. Scaling may never be used (i.e., 1x scaling), but again, it is dependent on what you are attempting to illustrate and highlight.

To answer your second question: this is again up to you and dependent on the problem. It depends on what you are attempting to highlight. You can print the data at various steps, and then visualize at each of those steps. In this way, you can see how deformation proceeds as a function of strain (and thus time).
